### PR TITLE
Upgrade Kuberntes to v1.15.2

### PR DESCRIPTION
**Pull request by masa213f**

### CODE REVIEW

**Comment by ymmt2005** [comment]
Please update etcd & etcdutil.

File: Makefile [hunk 1]
```diff
@@ -19,6 +19,10 @@ test:
 	go test -race -v ./...
 	go vet ./...
 
+static:
```
Comment (ymmt2005) [inline Makefile]: Add this to `.PHONY` targets, or this target does nothing because of `static` directory.

File: resource.go [hunk 2]
```diff
@@ -101,7 +100,6 @@ func init() {
 		runtime.NewMultiGroupVersioner(corev1.SchemeGroupVersion),
 		runtime.NewMultiGroupVersioner(policyv1beta1.SchemeGroupVersion,
 			schema.GroupKind{Group: policyv1beta1.GroupName},
-			schema.GroupKind{Group: extensionsv1beta1.GroupName},
```
Comment (ymmt2005) [inline resource.go]: This can be simpler.

**Comment by ymmt2005** [comment]
@masa213f @binoue 
Please fix the update procedure of `k8s.io/xxx` like this:

```console
$ go get k8s.io/client-go@kubernetes-1.15.2
```

**Comment by masa213f** [comment]
> @masa213f @binoue
> Please fix [the update procedure of k8s modules](https://github.com/cybozu-go/cke/blob/master/DEVELOP.md#update-k8sio-modules) like this:
> 
> ```
> $ go get k8s.io/client-go@kubernetes-1.15.2
> ```

@ymmt2005 
Thank you. client-go is already upgraded in cfcb7af.

Memo:
The hash of client-go@kubernetes-1.15.2 is [2fe0317bcee0](https://github.com/kubernetes/client-go/commit/2fe0317bcee0e1ff48861f77cc1e2791cceba458).

**Comment by ymmt2005** [comment]
@masa213f 
No, you should edit `DEVELOP.md`.
https://github.com/cybozu-go/cke/blob/master/DEVELOP.md#update-k8sio-modules

**Comment by masa213f** [comment]
> @masa213f
> No, you should edit `DEVELOP.md`.
> https://github.com/cybozu-go/cke/blob/master/DEVELOP.md#update-k8sio-modules

File: DEVELOP.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ specified:
 as follows:
 
 ```console
-$ go get k8s.io/client-go@v8.0.0
+$ go get k8s.io/client-go@kubernetes-1.15.2
 $ go get k8s.io/api@release-1.11
```
Comment (ymmt2005) [inline DEVELOP.md]: IIRC, this and `k8s.io/apimachinery` need not be updated manually
since `k8s.io/client-go` contains appropriate `go.mod`.

**Pull request merged**